app/screens/WalletNavigator/WalletNavigator.tsx

```diff
@@ -19,6 +21,14 @@ export interface WalletParamList {
     words: string[]
   }
   RestoreMnemonicWallet: undefined
+  PinCreation: {
```

Comment:
onboarding flow
- show words (carry forward: words)
- verify words (carry forward: words, desired pin length)
- pin creation (carry forward: words, created pin)
- pin verification (encrypt and save wallet)

Reply:
this will be further extended later
4. pin verification (carry forward encrypted wallet data, verified pin)
5. enroll biometric (store wallet, store pin into secure store)